### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by mslabek**

### CODE REVIEW

File: docs/Changelog.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 1.12.0 (Not released yet)
 -------------------------
+* [#4540](https://github.com/TouK/nussknacker/pull/4540) Improvement: Allow to select a claim from OIDC JWT to represent the username
```
Comment (lciolecki) [inline docs/Changelog.md:5]: Improvement: Allow selecting a claim from OIDC JWT to represent the username

File: docs/Changelog.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
...
+* [#4555](https://github.com/TouK/nussknacker/pull/4555) Remove: Back compatibility for encoding/decoding UIParameter
 * [#4561](https://github.com/TouK/nussknacker/pull/4561) Improvement: Users are not requrired at OAuth2 config file
 * [#4492](https://github.com/TouK/nussknacker/pull/4492) Allow to test fragments using ad hoc testing.
```
Comment (lciolecki) [inline docs/Changelog.md:8]: Allow testing

**Review by lciolecki — APPROVED** [review]

File: docs/Changelog.md (hunk 1 above)
Comment (mslabek) [reply docs/Changelog.md]: done

File: docs/Changelog.md (hunk 2 above)
Comment (mslabek) [reply docs/Changelog.md]: done

**Pull request merged**